amf-cli/jvm/src/test/scala/amf/client/validation/JvmPayloadValidationTest.scala

```diff
@@ -36,4 +35,23 @@ class JvmPayloadValidationTest extends ClientPayloadValidationTest with NativeOp
     positiveNumberReport.results.asSeq.head.message shouldBe "Can't divide by 0"
     zeroReport.results.asSeq.head.message shouldBe "Can't divide by 0"
   }
+
+  test("Validation order shouldn't affect conformance of each payload") {
+    val itemsShape = new NodeShape()
+    val shape      = new ArrayShape().withItems(itemsShape)
+    val validator  = payloadValidator(shape, `application/yaml`)
+
+    val validPayload   = "- {}"
```

Comment:
Was it only happening with syntax errors ?

Reply:
yes, it only seems to happen with syntax errors that reside in a multi-line payload